rca/project_styleguide/templates/patterns/organisms/programme_tabs/programme_overview.html

```diff
@@ -53,10 +53,15 @@ <h3 class="section__heading section__heading--secondary heading heading--four">
 </section>
 
 <section class="section bg bg--light">
-
-    {% include "patterns/organisms/staff-overview/staff-overview.html" %}
-
-    {% include "patterns/organisms/staff-overview/staff-overview--single.html" %}
+    {% with page.related_staff.all as staff %}
```

Comment:
Should we prefer the more modern format? https://docs.djangoproject.com/en/2.2/ref/templates/builtins/#with

`{% with staff=page.related_staff.all %}`